protocol/message.go

```diff
@@ -118,6 +201,12 @@ func NewKeyLookupInEpochProof(ap *m.AuthenticationPath,
 	}, e
 }
 
+// NewMonitoringProof creates the response message a CONIKS server sends
+// to a client upon a MonitoringRequest. The contained DirectoryProofs
+// includes a list of proofs of inclusion and a list
+// of STRs for the epoch range [startEpoch, endEpoch], where startEpoch
+// and endEpoch are the epoch range endpoints indicated in the client's
+// request. The error code is set to Success.
```

Comment:
> I think it would make sense to change the signature of the constructor and to set the ErrorCode internally to Success if that's the only "error" the constructor can ever return.

Yes, I agree.